libs/market-info/src/components/fees-breakdown/fees-breakdown.tsx

```diff
@@ -115,7 +115,7 @@ export const FeesBreakdown = ({
         </dd>
       )}
       <dd className="text-right col-span-2">
-        {formatValue(totalFees)} {quoteName || ''}
+        {formatValue(totalFees)} {symbol || ''}
```

Comment:
symbol or quoteName :/

Reply:
I am updating deal ticket price and liquidation to show quote name and I am leaving margin and fees in settlement asset symbol